submissions/israelOladosu/htmlForms/ContactForm.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <title>Contact Form Page</title>
+</head>
+
+<body>
+ <header>
+  <!-- container for the site logo -->
+  <div>
+   <a href="index.html">
+    <img src="./img/Farmville flat green.png" alt="jumia">
```

Comment:
I think your images should be in introductionToHtml folder. You can link the image to this file. Your htmlForm folder should only contain forms.